tfexec/apply.go

```diff
@@ -99,6 +100,30 @@ func (tf *Terraform) Apply(ctx context.Context, opts ...ApplyOption) error {
 	return tf.runTerraformCmd(ctx, cmd)
 }
 
+// ApplyJSON represents the Terraform apply subcommand with the `-json` flag.
```

Comment:
These new commands could likely use a link to https://developer.hashicorp.com/terraform/internals/machine-readable-ui in the Go documentation (or something to help consumers try to use it).

Reply:
Have linked to https://developer.hashicorp.com/terraform/internals/machine-readable-ui  in the Go docs.